src/system_tests/input_files/tMHDSYSTEMSodShockTubeParameterizedMpi_CorrectInputExpectCorrectOutput.txt

```diff
@@ -0,0 +1,57 @@
+#
```

Comment:
I changed the comment so it's marked as 3D now.

This is temporary. The current hydro sod shock tube test doesn't work with MHD since MHD doesn't have PPMP implemented yet, only PCM. I looked at generalizing the hydro test but it was more work than it was worth so I made a temporary test for the MHD version of the sod shock tube.